themes/bootstrap3/js/account_ajax.js

```diff
@@ -66,7 +67,7 @@ VuFind.register('account', function Account() {
         } else {
           $element.removeClass('hidden');
           if (status === LOADING) {
-            $element.html('<i class="fa fa-spin fa-spinner"></i>');
+            $element.html(VuFind.spinner());
```

Comment:
Shouldn't this change have been included in release 8.0, since we did all the spinner stuff there? I either missed it at the time or it was subsequently reintroduced. A quick search of the current dev branch for "fa-spinner" reveals a few instances. Should we open a separate PR to clean those up?

Reply:
https://github.com/vufind-org/vufind/pull/2274

Reply:
Thanks!